models/lido/lido_liquidity.sql

```diff
@@ -18,10 +18,16 @@
  ref('lido_liquidity_arbitrum_balancer_pools'),
  ref('lido_liquidity_optimism_balancer_pools'),
  ref('lido_liquidity_polygon_balancer_pools'),
+ ref('lido_liquidity_ethereum_balancer_pools'),
  ref('lido_liquidity_arbitrum_curve_pools'),
  ref('lido_liquidity_optimism_curve_pools'),
- ref('lido_liquidity_optimism_velodrome_pools'),
  ref('lido_liquidity_ethereum_curve_steth_pool'),
+ ref('lido_liquidity_ethereum_curve_steth_conc_pool'),
```

Comment:
minor: there is inconsistency with `pool` or `pools` usage in file names. don't worry about for now, let's keep as-is for spark, but consider this when migrating to dunesql if you'd like to keep that way or make consistent.